spyder/utils/programs.py

```diff
@@ -1062,3 +1062,24 @@ def find_git():
         return find_program('git')
     else:
         return find_program('git')
+
+
+def get_user_env_variables():
+    """Return local environment variables"""
+    cmdstr = ''
+    if sys.platform == 'darwin':
+        cmdstr = ('[[ -e /etc/profile ]] && source /etc/profile; '
+                  '[[ -e ~/.bash_profile ]] && source ~/.bash_profile; '
+                  'printenv')
+    elif sys.platform.startswith('linux'):
+        pass

```

Comment:
Linux should work very similar to Mac, right? Could we make this case work too?

Reply:
Absolutely! That is why I left a placeholder. Can you provide the profile files appropriate for linux?